browser/mocha/fixture.js

```
@@ -14,6 +14,10 @@ extendInterfaces('fixture', function(context, teardown) {
 
     // Find the test-fixture with the provided ID and create it, returning
     // the results:
-    return document.getElementById(fixtureId).create(model);
+    var fixture = document.getElementById(fixtureId);
+    if (!fixture) {
+      throw 'Could not find fixture with ID <' + fixtureId + '>'; 
```

can you throw an error instead of a string?

```
throw new Error('Could not find fixture with ID <' + fixtureId + '>');
```